CMSgov/mint-app
# [EASI-3618] Update Precedence Rules

# EASI-3618

## Changes and Description


Backend Changes from @OddTomBrooks 
- Updated gql schema
- Created flyway migration adding new enum type and table columns
- Updated data model
- Updated Postman collection


Frontend Changes from @garyjzhao 
- Updated Precendence Rules question to now have a checkbox and additional detail text input
- Updated `precendenceRules` in `src/i18n/en-US/modelPlan/beneficiaries.ts` to incorporate new YesNo options
- Added new 'No additional information specified' copy to `miscellaneous.ts`
- Update tests and snaps


## How to test this change

BE:
- Bring servers up
- Verify migration succeeded
- Run plan beneficiaries update endpoint in Postman
- Verify success and data in response

FE:
- Navigate to Frequency questions in Beneficiaries
- See changes have been made to the question
- Ensure data is recorded in read only and csv



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
